src/engine/TPA.cc

```diff
@@ -392,6 +392,11 @@ VerificationAnswer TPABase::solveTransitionSystem(TransitionSystem & system) {
 }
 
 VerificationAnswer TPABase::solve() {
+    auto res = checkTrivialUnreachability();
+    assert(res != VerificationAnswer::UNSAFE);
```

Comment:
Why even have this assert here?
It's unreachable in any case, because it is never returned from `checkTrivialUnreachability`

Reply:
At the moment, yes. It's just a check for some future person that would try to change something in the `checkTrivialUnreachability` method.